docs/zh/latest/FAQ.md

```diff
@@ -707,6 +707,21 @@ make GOOS=linux GOARCH=amd64
     prefix: "/apisix"
 ```
 
+## APISIX 如何配置带认证的 ETCD
+
+如果你使用的 ETCD 实例，是需要使用正确的用户和密码才能访问。你需要在`conf/config.yaml`文件中这样配置。
```

Comment:
Update the sentence as per the English ones.

Reply:
done